kubejs/server_scripts/enigmatica/kubejs/expert/unification/unify_materials.js

```diff
@@ -282,8 +282,8 @@ onEvent('recipes', (event) => {
                 output: Ingredient.of(levigated_material).toJson(),
                 catalyst: Ingredient.of('naturesaura:crushing_catalyst').toJson(),
                 aura_type: 'naturesaura:overworld',
-                aura: 1500,
-                time: 10
+                aura: 300,
```

Comment:
I believe your intention was to increase this?

Reply:
yep, it's doubled. was 150 per tick, now it's 300 per tick

Reply:
er, wait where'd the time change go >.>

Reply:
ok, no time is still 1, so 300 per tick :D

Reply:
The old value was 1500 per tick though, according to the diff above?

Reply:
1500 over ten. It's the total cost, not per tick cost

Reply:
Ahh okay, thanks 😅